rhel/help.md

```diff
@@ -0,0 +1,143 @@
+% IMAGE_NAME(1)
+% MAINTAINER
+% DATE
+
+# DESCRIPTION
+
+The Datadog Agent provides real-time performance tracking and visualization of your containers, orchestrators,
+operating system and application metrics.
+
+# SUPPORT
+
+Datadog's support team is available to assist you with any Datadog Agent related issues.
+Support can be reached via [ticket](http://docs.datadoghq.com/help/), chat or other channels.
+
+Please remember to [send your logs](https://github.com/DataDog/dd-agent/wiki/Send-logs-to-support)
```

Comment:
blank page - let's use https://help.datadoghq.com/hc/en-us/articles/204991415-Send-logs-and-configs-to-Datadog-via-flare-command instead?